mod-source-record-manager-server/src/main/java/org/folio/services/JobExecutionsCache.java

```diff
@@ -40,15 +41,15 @@ public JobExecutionsCache(JobExecutionService jobExecutionService, Vertx vertx,
     cache = buildCache();
   }
 
-  public Future<JobExecutionDtoCollection> get(String tenantId, String cqlQuery, int offset, int limit) {
+  public Future<JobExecutionDtoCollection> get(String tenantId, JobExecutionFilter filter, String sortBy, String order, int offset, int limit) {
```

Comment:
@RuslanLavrov  Do we have smth like validation for this JobExxecutionFilter?

Reply:
we have validation for Date parameters is that performed by RMB before call our endpoint